tests/ui/wildcard_enum_match_arm.rs

```diff
@@ -1,7 +1,13 @@
 // run-rustfix
 
 #![deny(clippy::wildcard_enum_match_arm)]
-#![allow(unreachable_code, unused_variables, dead_code, clippy::single_match)]
+#![allow(
+    unreachable_code,
+    unused_variables,
+    dead_code,
+    clippy::single_match,
+    clippy::pats_with_wild_match_arm
```

Comment:
This shouldn't be required once the check for `non_exhaustive` is added.

Reply:
I can't remove this, because using `cargo test` (that uses `-D warning` flag) means that I get an error. Do you have any tip on this ?